wikimedia/wikipedia-ios
# Image recommendations - Go to article preview

**Phabricator:** https://phabricator.wikimedia.org/T358920

### Notes
* This PR adds the ability to go from the InsertMediaSettingViewController to the EditPreviewViewController when coming from the image recommendations feature
* It depends on https://github.com/wikimedia/wikipedia-ios/pull/4800

### Test Steps
1. Run the experimental app
2. Tap the Image Recommendation card
3. Tap "Yes" for a recommendation
4. On the Add Image Details view, add some caption and tap next
5. You should preview the first section of the article with the image and caption
6. Go to the page editor, add an image and preview the edit, make sure nothing breaks or looks off
